storybook/native/main.js

```diff
@@ -31,6 +32,8 @@ addDecorator((Story) => (
   </SpaceAround>
 ));
 
+addDecorator(withKnobs);
```

Comment:
could you add it in the preview Js too for web

Reply:
Done